helmfile.d/values/opensearch-op/ingress.yaml.gotmpl

```
@@ -0,0 +1,45 @@
+cluster:
+  ingress:
+    opensearch:
+      {{- if not .Values.opensearch.clientNode.dedicatedPods }}
+      enabled: true
```

This should fix the test failures
```suggestion
      {{- if .Values.opensearch.clientNode.dedicatedPods }}
      enabled: false
      {{- else }}
      enabled: true
```